web_monitoring/pf_edgi.py

```diff
@@ -20,6 +20,17 @@ def list_cabinets():
     assert content['status'] == 'ok'  # business logic is OK
     return content['cabinets']
 
+def get_cabinetID(url):
+    cabinets = list_cabinets()
+    cabinetID = ''
+    for key,value in cabinets.items():
+        for dictionary in value:
+            if(str(urlparse(dictionary['url']).scheme + '://' + urlparse(dictionary['url']).netloc) == url):
+                cabinetID = dictionary['name']                
+
+    assert cabinetID != ''
```

Comment:
Could we raise a `ValueError` with a message (something like "no such url") instead of a bare `AssertionError`?